db/migrate/20160816000015_create_teachers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTeachers < ActiveRecord::Migration
+  def change
+    create_table :teachers do |t|
+      t.string :name
+      t.string :bio
+      t.string :certifications
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
You got the attributes right and this looks good, but I don't see an updated schema file, make sure you rake db:migrate before you push your branch. Go ahead and run rake db:migrate and then push up your changes